phoenix-core-client/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1634,7 +1636,9 @@ public MutationState createIndex(CreateIndexStatement statement, byte[][] splits
 
         Map<String,Object> tableProps = Maps.newHashMapWithExpectedSize(statement.getProps().size());
         Map<String,Object> commonFamilyProps = Maps.newHashMapWithExpectedSize(statement.getProps().size() + 1);
-        populatePropertyMaps(statement.getProps(), tableProps, commonFamilyProps, PTableType.INDEX);
+        populatePropertyMaps(statement.getProps(), tableProps, commonFamilyProps, PTableType.INDEX,
+                CDCUtil.isCDCIndex(SchemaUtil
+                        .getTableNameFromFullName(statement.getIndexTableName().toString())));
```

Comment:
Do we need this special handling considering `createCDC` is explicitly adding `REPLICATION_SCOPE=0` property? We need only one of these right?